docs/architecture/modernize-desktop/deploying-modern-desktop-applications.md

```diff
@@ -0,0 +1,332 @@
+MSI, ClickOnce, App-V, and scripting. Each of  technologies has their strengths and
+Microsoft has decided to pick the best of all to build MSIX. Therefore, MSIX is
+built on the foundations of these existing technologies picking the best of
+each:
+
+* App-V =\> Containerization
+* ClickOnce =\> Auto updating
+* MSI =\> Easy to distribute
+
+With MSIX, we get one installer technology with all these features.
+
+![MSIX](media\deploying-modern-desktop-applications\msix.png)
+
+### Benefits of MSIX
+
+#### Never regret installing an App
+
+MSIX provides a predictable, reliable, and safe deployment. The declarative
+method contained in the package manifest lets the OS keep track of every asset
+your application needs. It also provides a true clean uninstall with no side
+effects.
+
+#### Disk space optimization
+
+MSIX is optimized to reduce the footprint and application has on the user's
+machine disk space. It creates a single instance storage of your files, meaning
+that if you have two different packages with the same DLL, it is not installed
+twice. The platform takes care of that because it knows all the files that are
+installed by a particular app thanks to its declarative nature. It also allows
+you to have different versions of a DLL working side by side.
+
+With the use of Resource packages, you can easily create Multilingual apps and
+the OS takes care of installing the ones that are used.
+
+#### Network optimization
+
+MSIX detects the differences on the files at the byte block level enabling a
+feature called Differential Updates. What this means it that only the updated
+byte blocks are downloaded on application updates.
+
+![MSIX managing updates](media\deploying-modern-desktop-applications\msix-managing-updates.png)
+
+With Streaming installation, the user can start working on your application
+quickly while other parts of the app are downloaded on the background.
+This feature contributes to an engaging experience for your users.
+
+Through optional packages feature you achieve componentization on your app
+deployment, so you can download them when needed.
+
+#### Simple packaging and deployment
+
+The AppManifest declares the versioning, device targeting and identify in a
+standard way for every application. It also provides a way to sign your assets
+providing a solid security foundation.
+
+#### OS Managed
+
+The OS handles all the processes for installation, updating and removing an
+application. Applications are installed per user but downloaded only once,
+minimizing the disk footprint. Microsoft is working on providing the MSIX
+experience also on Windows 7.
+
+#### Windows provides integrity for the app
+
+With the use of digital signatures, you can guarantee that you don't install
+application from untrusted sources. MSIX also prevents tampering because it
+keeps a record of file hashes and detects on file has been modified after
+installation.
```

Comment:
```suggestion
With the use of digital signatures, you can guarantee that you don't install an
application from untrusted sources. MSIX also prevents tampering because:

- It keeps a record of file hashes.
- It detects if a file has been modified after installation.
```